src/adaptors/adaptor_common.c

```diff
@@ -159,8 +159,8 @@ int qd_raw_connection_grant_read_buffers(pn_raw_connection_t *pn_raw_conn)
     //
     // Note also that the stats pointer may be NULL if no buffers have yet been allocated.
```

Comment:
This comment should be modified or removed ?

Reply:
Yes good find, will fix.